extensions/vertx-common/deployment/src/main/java/io/quarkus/vertx/common/deployment/InternalVertxBuildItem.java

```diff
@@ -0,0 +1,19 @@
+package io.quarkus.vertx.common.deployment;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+import io.quarkus.runtime.RuntimeValue;
+import io.vertx.core.Vertx;
+
+public final class InternalVertxBuildItem extends SimpleBuildItem {
```

Comment:
If we use `Common` in the artifactId, shouldn't we say `CommonVertxBuildItem`.

Actually, as it's the shared instance, we could rename that to `vertx-core`.

Reply:
I renamed it to `CoreVertxBuildItem`.